static/js/fm-zone.js

```diff
@@ -241,7 +241,7 @@
 			$( this ).fm_zonify( posts );
 		}
 
-		$( '.fm-zone-posts-wrapper:visible' ).each( zonifier );
+		$( '.fm-zone-posts-wrapper' ).each( zonifier );
```

Comment:
I went down this road with Fieldmanager a while ago, and this was my first solution, also. However, I ended up implementing a more robust solution that a) moves scripts to the footer and b) uses Gutenberg's DOM ready helper. That PR is here:

https://github.com/alleyinteractive/wordpress-fieldmanager/pull/757

I believe that the configuration of fm-zones is much simpler compared to Fieldmanager itself, so integration here would likely be significantly more straightforward. The important parts are:

* Ensure the fm-zones JavaScript is being enqueued in the footer.
* Create a function like `fmLoadModule` in the PR above and use it in place of `$(document).ready` (https://github.com/alleyinteractive/wordpress-fieldmanager/pull/757/files#diff-92affb4a9da5be89b7b8cf47a3822c1eba297ee97c9f7487feaea5f214726c45). Note that the function should not conflict with the Fieldmanager version in case they are both loaded at the same time.
* Ensure that any scripts (like this one) that depend on the loader are loaded after the loader, either by ensuring the loader is defined in the file that needs it, or if there are multiple JavaScript files that need it, it is broken out into its own file and loaded separately and then listed as a dependency for the other files (this is what Fieldmanager is doing, due to the number of files that need to be loaded).